vermouth/data/force_fields/elnedyn22/modifications.ff

```diff
@@ -22,3 +22,8 @@ BB {"replace": {"atype": "P5", "charge": 0}}
 NH2-ter
 [ atoms ]
 BB {"replace": {"atype": "P5", "charge": 0}}
+
+[ modification ]
+ASP0
+[ atoms ]
+SC1 {"resname": "ASP", "replace": {"atype": "P3", "charge": 0}}
```

Comment:
Is it clever to call it ASP0? I recall from talks with paulo that it should be AS0. In any case at the moment we use up to three or four naming convention for neutral residues. I would argue before merging we should unify these.

Reply:
These are modifications, which do not end up in the PDB as resname (although it's possible to rename e.g. res ASP + mod ASP-H -> res AS0, should we so desire).
I suggest we decide on a naming convention at a later date, and leave it for now.